helix-core/src/text_annotations.rs

```diff
@@ -154,6 +154,15 @@ pub struct TextAnnotations {
 }
 
 impl TextAnnotations {
+    // Create a set of empty TextAnnotations
```

Comment:
You can use `TextAnnotations::default` here (which is what is used elsewhere to create a new TextAnnotations instance) so no need to add a new function